xbmc/input/MouseStat.cpp

```diff
@@ -231,7 +234,9 @@ void CMouseStat::SetActive(bool active /*=true*/)
   // 1. The mouse is active (it has been moved) AND
   // 2. The XBMC mouse is disabled in settings AND
   // 3. XBMC is not in fullscreen.
-  g_Windowing.ShowOSMouse(m_mouseState.active && !IsEnabled() && !g_Windowing.IsFullScreen());
+  CWinSystemBase &winSystem = CServiceBroker::GetWinSystem();
+  if (&winSystem)
```

Comment:
reference cannot be bound to dereferenced null pointer in well-defined C++ code; pointer may be assumed to always convert to true [-Wundefined-bool-conversion]

I think you already have to handle the null pointer case in `CWinSystemBase &CServiceManager::GetWinSystem()` if it can even happen.

Reply:
this is a left-over and will disappear soon when I'll move MouseStat from InputManager to WinSystem